package.json

```diff
@@ -46,20 +46,20 @@
     "yargs": "^17.7.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.9",
+    "@babel/core": "^7.22.10",
     "@codemod-utils/tests": "^0.2.5",
     "@sondr3/minitest": "^0.1.1",
     "@tsconfig/esm": "^1.0.4",
     "@tsconfig/node16": "^16.1.0",
     "@tsconfig/strictest": "^2.0.1",
-    "@types/node": "^16.18.39",
+    "@types/node": "^16.18.40",
     "@types/yargs": "^17.0.24",
-    "@typescript-eslint/eslint-plugin": "^6.2.1",
-    "@typescript-eslint/parser": "^6.2.1",
+    "@typescript-eslint/eslint-plugin": "^6.3.0",
+    "@typescript-eslint/parser": "^6.3.0",
     "concurrently": "^8.2.0",
     "eslint": "^8.46.0",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-import-resolver-typescript": "^3.5.5",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
The main change seemed to be supporting the flat configuration that will be available in `eslint@v9`.

https://github.com/prettier/eslint-config-prettier/blob/v9.0.0/CHANGELOG.md#version-900-2023-08-05